src/pages/RegisterPage.js

```diff
@@ -1,21 +1,42 @@
 import React from 'react';
 import { Grid, Row, Col } from 'react-flexbox-grid';
+import { reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { getTranslator } from '../reducers/translation';
 import HeaderRegistration from '../components/HeaderRegistration';
+import { SquareButton } from '../components/Buttons';
+import TextField from '../components/FieldText';
 import Footer from '../components/Footer';
-
 import common from '../styles/common.scss';
+import RegisterPageStyle from './RegisterPage.scss';
+import { LoginLink } from '../components/PredefinedLinks';
 
-export default () =>
+export const RegisterFormView = ({ handleSubmit, i18n }) =>
   (<div className={common.layoutAppContainer}>
     <HeaderRegistration />
     <div className={`${common.layoutContentArea} ${common.gradientContentArea}`}>
       <div className={common.layoutWidthLimiter}>
-
         <Grid fluid>
           <Row>
             <Col lgOffset={3} lg={6}>
               <div className={common.whiteArea}>
```

Comment:
Is it possible to apply `common.whiteArea` directly on the `Col` and skip a `div`?